core/src/scss/utilities/mixins/icons-logos/_icon.scss

```diff
@@ -0,0 +1,27 @@
+@charset "UTF-8";
+
+///
+/// Display an icon after a link.
```

Comment:
Reminder to change the description of this mixin :)

Reply:
Whoops! Fixing.